charts/apps/volumes-web-app/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
```

Comment:
bump chart versions as well for these apps as well?

Reply:
bumped chart versions for admission weboook, jupyter, notebook controller